.swiftlint.yml

```diff
@@ -112,6 +112,7 @@ excluded: # paths to ignore during linting. Takes precedence over `included`.
   - UITests/
   - XCUITests/
   - l10n-screenshots-dd/
+  - DerivedData/
```

Comment:
In this folder and in concrete in SourcePackages/checkouts there were some failures related to the new rules added. Since we can't modify those, I have added the path to ignored paths